stream_chat/tests/test_client.py

```diff
@@ -9,7 +9,6 @@
 from stream_chat.base.exceptions import StreamAPIException
 
 
-@pytest.mark.incremental
```

Comment:
since passes, looks like a good idea

Reply:
you can't use `@pytest.mark.xfail` with this

Reply:
could you put a link here for docs?